# li=[10,20,30,40]
# try:
#     print(li[5])
# except Exception as e:
#     print(e)


# try:
#     x=10/0
# except Exception as e:
#     print(e)
# print("hello")

# li=[10,20,30,40]
# try:
#     print(10/0)
#     print(li[5])
# except Exception as e:
#     print(e)
# print("helllo")


# li=[10,20,30,40]
# x=10
# try:
#     print(x/2)
#     print(li[3])
#     print(x/0)
# except Exception as e:
#     print(e)
# except:
#     print("an error occured")
# print("hello")


# try:
#     x = int(input("enter a number:"))
#     print(x)
#     y=[10,20,30]
#     print(y[1])
#     print(y[2])
#     print(y[3])
#     print(y[4])
# except Exception as a:
#     print(a)


# y=[10,20,30]
# x=10
# try:
#     print(y[0])
#     print(x/0)
#
# except IndexError: # Named Exception
#     print("trying to access non existent index")
# except ZeroDivisionError: #Named Exception
#     print("trying divide no by 0")
# except:# generic exception
#     print("An error occurred")




